src/App.tsx

```diff
@@ -15,47 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type Goods = string[];
+
+const SORT_FIELD_BY_LETTERS = 'letters';
+const SORT_FILED_BY_LENGTH = 'length';
```

Comment:
Since we're already using TypeScript here, it's worth storing it via `enam`, adding a default value (empty string) as well.